doc/implementation_comparison.md

```diff
@@ -139,7 +144,7 @@ https://github.com/grpc/grpc/blob/master/doc/compression_cookbook.md
 
 Grpc.Core: supported (exposed publicly on Channel), provided by C-Core
 
-grpc-dotnet: not supported (to provide support, changes to .NET HttpClient are needed, so adding support is non-trivial).
+grpc-dotnet: supported
```

Comment:
nit: does this require .NET5+ or .NET6+?  Might be worth documenting here. Same for wait-for-ready.

Reply:
```suggestion
grpc-dotnet: Supported on .NET 5 or later
```